lib/backend/watchersyncer/watchersyncer_test.go

```diff
@@ -131,21 +131,30 @@ var _ = Describe("Test the backend datastore multi-watch syncer", func() {
 		By("Driving a bunch of List complete, Watch fail events for 2/3 resource types")
 		expectedDuration := watchersyncer.WatchPollInterval * 2
 		minDuration := 70 * expectedDuration / 100
-		maxDuration := 130 * expectedDuration / 100
+		maxDuration := 140 * expectedDuration / 100
+		before := time.Now()
```

Comment:
Move this up since at the original location all the events will have been handled while expecting the status updates.